lib/_stream_writable.js

```diff
@@ -248,10 +248,18 @@ Writable.prototype.pipe = function() {
 
 
 function writeAfterEnd(stream, cb) {
-  const er = new ERR_STREAM_WRITE_AFTER_END();
-  // TODO: defer error events consistently everywhere, not just the cb
-  errorOrDestroy(stream, er);
-  process.nextTick(cb, er);
+  // TODO: defer error events consistently everywhere, not just the cb.
+  if (stream._writeAfterEnd) {
```

Comment:
```suggestion
  if (typeof stream._writeAfterEnd === 'function') {
```

Reply:
Why did you resolve this?

Reply:
I think I made a mistake somewhere. I thought I applied the suggestion.